app/dev-portal/portals/customization/snippets.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Snippets
+---
+
+Snippets enable reuse of content and MDC components across [Pages](/dev-portal/portals/customization/custom-pages). Visibility controls and Publishing status allow you to enable/disable Snippets, and/or restrict access to logged-in Developers.
+
+To get started creating Snippets, navigate to your Dev Portal and select [**Portal Editor**](/dev-portal/portals/customization/portal-editor/), then Snippets from the left sidebar.
+
+{:.note}
+> *Snippets are built using Markdown Components (MDC). Additional documentation on Snippet syntax and usage is available on a [dedicated MDC site](https://portaldocs.konghq.com/components/snippet).*
+
+
+### Create a new Snippet
+* Click `+` at the top
+* Give the Snippet a "name". This will be used to refer to your Snippet from Pages.
+* The Snippet will be created with the `title` in front matter set to the specified name. Snippet front matter is not used in generating Pages, but can be useful to keep track of what you're working on alongside other Editors.
+* Edit the content of your Snippet in the middle panel using any Markdown or MDC, and you'll see a Live Preview in the right panel.
+
+{:.important}
+> *Preview may not always be able to display parameterized values. When the Page is being rendered in your Portal, parameters will be resolved. Depending on the type of syntax used, the Preview may not reflect parameterized values in Page or Snippet views.*
+
+{:.note}
+> *Snippets are `Published` by default for ease of use. Unilt the Snippet has been referred to in a Page, it will not be displayed in the rendered portal.*
```

Comment:
typo
```suggestion
> *Snippets are `Published` by default for ease of use. Until the Snippet has been referred to in a Page, it will not be displayed in the rendered portal.*
```